python/src/wavelet_image.cc

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020-2022 PANDA GmbH
```

Comment:
Everything is ok, only organization of files. Lets put all .cc for WaveletImage to .img.